server/schedule/operator.go

```diff
@@ -104,6 +105,31 @@ func (rp RemovePeer) Influence(opInfluence OpInfluence, region *core.RegionInfo)
 	from.RegionCount--
 }
 
+// MergeRegion is an OperatorStep that merge two regions.
+type MergeRegion struct {
+	FromRegion uint64
+	ToRegion   uint64
+	Direction  pdpb.MergeDirection
+	IsFake     bool
```

Comment:
What's this used for?

Reply:
there are two regions involved in merge process, so to keep them from other scheduler, both of them should add operator. But actually, tikv just need one region to get the merge request, thus use a IsFake mark to indicate that this region doesn't need to send merge request to tikv

Reply:
I don't think `IsFake` is reasonable name here, maybe `IsPassive`. Also need some comments here.